### PR TITLE
Update README.md and example models.

### PR DESCRIPTION
**Pull request by milsman2**
- Demo model of breweries.
- Demo schema of breweries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ The Open Brewery DB API server is a FastAPI REST-based API server connected to a
   - `pytest`
 - Run the server
   - `uvicorn main:app`
+- Change the host to any connection
```
Comment (chrisjm) [inline README.md]: Let's nest these under "Run the server" for clarity. 

Also, feel free to completely reorg this section. I just kind of made it on the fly.

File: schemas/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+"""
+Import schemas to allow for module level access
+"""
+from app.schemas.breweries import  Brewery, BreweryCreate, BreweryUpdate, BreweryUpdateRestricted
```
Comment (chrisjm) [inline schemas/__init__.py]: Only `Brewery` is necessary. We're going to keep the API read-only for now.

File: models/brewery.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    address_3 = Column(String)
+    city = Column(String)
+    state = Column(String)
+    county_province = Column(String)
+    postal_code = Column(String)
+    website_url = Column(String)
+    phone = Column(String)
+    country = Column(String)
+    longitude = Column(Float)
+    latitude = Column(Float)
```
Comment (chrisjm) [inline models/brewery.py]: Let's also add `created_at` and `updated_at` columns. Datetime, whatever that is in Python :)

File: schemas/breweries.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    address_3: str | None
+    city: str | None
+    state: str | None
+    county_province: str | None
+    postal_code: str | None
+    website_url: str | None
+    phone: str | None
+    county: str | None
+    longtitude: int | None
+    latitude: int | None
```
Comment (chrisjm) [inline schemas/breweries.py]: Same `created_at` and `updated_at`

File: schemas/breweries.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    class Config:
+        orm_mode = True
+
+# Additional properties stored in DB but not returned by API
+class BreweryInDB(BreweryInDBBase):
+    ...
+
+# Additional properties to return via API
+class Brewery(BreweryInDBBase):
+    ...
```
Comment (chrisjm) [inline schemas/breweries.py]: Currently, it's a read-only API, so some of these classes aren't necessary. 

Not sure about the `BreweryInDB` classes. We don't store contributors in the DB yet but should be built in for v2.

**Review by chrisjm — CHANGES_REQUESTED** [review]
Thanks for pushing this forward @milsman2 ! Couple of comments and update requests. 

As always, feel free to question and/or push-back. 🙂🍻

**Comment by chrisjm** [comment]
Also, do you know if there are any tests we can add to this?

**Review by chrisjm — APPROVED** [review]

**Comment by milsman2** [comment]
There are definitely ways to test but I haven't gotten into that part of FastAPI too deep yet. Definitely can do soon down the road once we have an MVP.

**Pull request merged**